bubble_sort.rb

```diff
@@ -0,0 +1,32 @@
+def bubble_sort(arr)
+  has_changed = true
+
+  while has_changed == true
```

Comment:
Your `has_changed` variable is already boolean, there's no need testing for `has_changed == true`, the code below would work as fine
```
while has_changed
    # your code here
end
```